src/hooks/spaces/useSpaceBySlug.ts

```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+import { useFirestore, useFirestoreCollectionData } from "reactfire";
+import Bugsnag from "@bugsnag/js";
+
+import { AnyVenue } from "types/venues";
+
+import { withIdConverter } from "utils/converters";
+import { WithId } from "utils/id";
+
+type UseSpaceBySlugResult = {
+  space?: WithId<AnyVenue>;
+  isLoaded: boolean;
+};
```

Comment:
```suggestion
  error?: string;
};
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2548/commits/9cc14823a26b81030e32d035218556fed2c2c758